XSConsoleTerm.py

```diff
@@ -283,6 +283,8 @@ def MainLoop(self):
             # if gotKey is not None:
             #     bannerStr = gotKey
 
+            # Reinitialize the time conversion rules from the system timezone
+            time.tzset()
```

Comment:
Wouldn't it be better to call `time.tzset()` after the timezone is actually changed rather than every time we go round the main loop?

Reply:
Hi @rosslagerwall , I updated as you mentioned and tested successfully. could you review this changes?